lib/macho/load_commands.rb

```diff
@@ -111,7 +111,7 @@ module LoadCommands
       # "reserved for internal use only", no public struct
       :LC_PREPAGE => "LoadCommand",
       :LC_DYSYMTAB => "DysymtabCommand",
-      :LC_LOAD_DYLIB => "DylibCommand",
+      :LC_LOAD_DYLIB => "DylibUseCommand",
```

Comment:
Did this and moved the `new_from_bin` hack to there to be slightly more strict where this applies - it should not apply to `LC_ID_DYLIB` etc.